packages/file_selector/file_selector_macos/CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
-## NEXT
+## 0.9.3+4
 
+* Remove deprecated `primary` and `onPrimary` references from `ElevatedButton.styleFrom`.
```

Comment:
These changes don't need changelog or version entries; they are to non-main example files so fall under the second exemption.

Reply:
Thanks a lot for helping review! I just reverted the change for changelog and the version entires!